db/migrate/20210924102745_add_index_on_blood_pressures_facility_id.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexOnBloodPressuresFacilityId < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    add_index :blood_pressures, [:facility_id], name: :index_blood_pressures_on_facility_id, algorithm: :concurrently
```

Comment:
one question I realized after approving / merging -- what will this do in production with the index you created already out there?

Reply:
Looks like it just added it again on IN prod? 

https://simple.semaphoreci.com/jobs/1aa42995-05b2-4ecc-9581-7d4aa52acf45

```
00:42 deploy:migrating_with_data 00:46
824      01 $HOME/.rbenv/bin/rbenv exec bundle exec rake db:migrate:with_data 00:48
825      01 == Schema ===================================================================== 00:48
826      01 == 20210924102745 AddIndexOnBloodPressuresFacilityId: migrating =============== 00:48
827      01 -- add_index(:blood_pressures, [:facility_id], {:name=>:index_blood_pressures_on_facility_id, :algorithm=>:concurrently}) 00:53
828      01    -> 4.9486s 00:53
829      01 == 20210924102745 AddIndexOnBloodPressuresFacilityId: migrated (4.9487s) ====== 00:53
830      01 
```

Reply:
Yeah, it's okay to have duplicate indexes. I have now dropped the temporary one.